app/frontend/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -36,11 +73,20 @@ export default function GroupChatView() {
         <CloseIcon />
         (leave)
       </Button>
-      {error && <Alert severity="error">{error}</Alert>}
-      {loading ? (
+      {(error || sendMutation.error || leaveGroupChatMutation.error) && (
```

Comment:
`error` should exclude the loading/data UI from rendering below if there is an error, so we know `messages` would exist.

The `sendMutation` and `leaveGroupChatMutation` can then share a separate error message since we probably don't want to hide the messages because of either mutation errors.

Reply:
I don't think that's the case, there might be an error fetching but messages shows the cached data.

Reply:
Actually, that applies to the other ones as well

Reply:
See latest commit, `error` can exist while not loading but still having old data to show. So I just tested to make sure the array exists before passing it to the child "...View" component.

Reply:
Ahh yeah for the refetched error case. I guess the types were right all along for us to be cautious in the first place.